<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QueueControlButtonsWidget</class>
 <widget class="QWidget" name="QueueControlButtonsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>18</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>222</width>
    <height>17</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>320</width>
    <height>18</height>
   </size>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="0">
        <red>239</red>
        <green>240</green>
        <blue>241</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="0">
        <red>239</red>
        <green>240</green>
        <blue>241</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="0">
        <red>239</red>
        <green>240</green>
        <blue>241</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="0">
        <red>239</red>
        <green>240</green>
        <blue>241</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="queueControlsContainer" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>222</width>
       <height>17</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>320</width>
       <height>18</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QToolButton" name="addButton">
        <property name="minimumSize">
         <size>
          <width>22</width>
          <height>18</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>22</width>
          <height>18</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#addButton {
	background-color: transparent;
	border-image: url(:assets/qadd.png);
	background: none;
	background-repeat: none;
}

QToolButton::menu-indicator { image: none; }

#addButton:pressed {
	border-image: url(:assets/qadd_file_p.png);
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacerAdd">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>18</width>
          <height>1</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="removeButton">
        <property name="minimumSize">
         <size>
          <width>22</width>
          <height>18</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>22</width>
          <height>18</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#removeButton {
	background-color: transparent;
	border-image: url(:assets/qrem.png);
	background: none;
	background-repeat: none;
}

QToolButton::menu-indicator { image: none; }

#removeButton:pressed {
	border-image: url(:assets/qrem_sel_p.png);
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacerRemove">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>18</width>
          <height>1</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="selectButton">
        <property name="minimumSize">
         <size>
          <width>22</width>
          <height>18</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>22</width>
          <height>18</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#selectButton {
	background-color: transparent;
	border-image: url(:assets/qsel.png);
	background: none;
	background-repeat: none;
}

QToolButton::menu-indicator { image: none; }

#selectButton:pressed {
	border-image: url(:assets/qsel_all_p.png);
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacerSelect">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>18</width>
          <height>1</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="miscButton">
        <property name="minimumSize">
         <size>
          <width>22</width>
          <height>18</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>22</width>
          <height>18</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#miscButton {
	background-color: transparent;
	border-image: url(:assets/qmisc.png);
	background: none;
	background-repeat: none;
}

QToolButton::menu-indicator { image: none; }

#miscButton:pressed {
	border-image: url(:assets/qmisc_opts_p.png);
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::MinimumExpanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>6</width>
          <height>1</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="optionsButton">
     <property name="minimumSize">
      <size>
       <width>22</width>
       <height>16</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>22</width>
       <height>16</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#optionsButton {
	background-color: transparent;
	border-image: url(:assets/qlistopts.png);
	background: none;
	background-repeat: none;
}

QToolButton::menu-indicator { image: none; }

#optionsButton:pressed {
	border-image: url(:assets/qlistopts_load_p.png);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
       <spacer name="horizontalSpacerOptions">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>36</width>
          <height>1</height>
         </size>
        </property>
       </spacer>
      </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
